packages/web/src/pages/upload-page/fields/AccessAndSaleField.tsx

```diff
@@ -159,7 +161,8 @@ export const AccessAndSaleFormSchema = (trackLength: number) =>
     .refine(
       (values) => {
         const formValues = values as AccessAndSaleFormValues
-        if (isPremiumContentUSDCPurchaseGated(formValues[PREMIUM_CONDITIONS])) {
+        const isUsdcGated = formValues[AVAILABILITY_TYPE] === 'USDC_PURCHASE'
```

Comment:
I would be okay with these just being inside the conditionals but no big deal.

Reply:
will do